src/js/themes/responsive.js

```diff
@@ -13,19 +13,14 @@ import { MediaThemeElement } from '../media-theme-element.js';
 const template = document.createElement('template');
 template.innerHTML = `
 <style>
-  :host {
-    display: inline-block;
-  }
-  media-controller:not([audio]) {
-    display: block;
-    aspect-ratio: 16 / 9;   /* set container aspect ratio if preload=none */
```

Comment:
I guess this moved to the html?

Reply:
yup, this should be handled by the consumer of the theme